lib/resty/waf.lua

```diff
@@ -209,7 +209,7 @@ local function _do_transform(self, collection, transform)
 				return collection -- dont transform if the collection was nil, i.e. a specific arg key dne
 			end
 
-			--_LOG_"doing transform of type " .. transform .. " on collection value " .. tostring(collection)
+			if self._debug == true then ngx.log(self._debug_log_level, '[', self.transaction_id, '] ', "doing transform of type " .. transform .. " on collection value " .. tostring(collection)) end
```

Comment:
Did the makefile make all these changes here with the debug stuff?

Reply:
Yeah

Reply:
I wasn't certain whether I should commit it or not

Reply:
I don't think we'd ever enable debug logging on the waf, unless you think we'd want that capability for testing with this new build? 

It'd really just be a micro-optimisation to skip checking the `_debug` property.

Thoughts?

Reply:
:+1: Have removed these changes